spidermon/contrib/validation/cerberus/validator.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import
+from cerberus.validator import (
+    Validator as cerberus_validator,
+    DocumentError,
+    SchemaError,
+)
+from spidermon.contrib.validation.validator import Validator
+from .translator import CerberusMessageTranslator
+
+
+class CerberusValidator(Validator):
+    default_translator = CerberusMessageTranslator()
+    name = "Cerberus"
+
+    def __init__(self, schema, translator=None, use_default_translator=True):
+        super(CerberusValidator, self).__init__(
+            translator=translator, use_default_translator=use_default_translator
+        )
+        self._schema = schema
+
+    def _validate(self, data, strict=False):
+        try:
+            validator = cerberus_validator(self._schema)
+            validator.validate(data)
+            for field_name, error in validator.errors.items():
+                self._add_errors({field_name: error})
+        except DocumentError:
+            raise ValueError("{} is not a document, must be of dict type".format(data))
```

Comment:
According to [Cerberus documentation](https://docs.python-cerberus.org/en/stable/schemas.html#validation-schemas), a valid schema is a [mapping](https://docs.python.org/3/glossary.html#term-mapping), so the error message should comply with this rule to avoid confusion.

Reply:
Checking if object is an instance of `collections.abc.Mapping` should be enough to verify if the schema provided has the right format (not that it is a valid schema).

Reply:
Now, that I check out `Mapping`, I realize this is actually a better way to go!
Again, should I just change the error message (Let Cerberus check it) or should we bypass with our own method which is kinda the same way

I am going with changing the message, to something more simpler and easy to understand.